test/regress/output/jobench/1a.txt

```diff
@@ -0,0 +1,49 @@
+SELECT MIN(mc.note) AS production_note,
```

Comment:
You can add a gitkeep file there like this: https://github.com/zhouqingqing/qpmodel/blob/master/test/regress/output/tpch1/.gitkeep